### PR TITLE
#84 - Added Koa middleware support

### PR DESCRIPTION
**Pull request by samarthmaniar**
Fix for #84

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -5,5 +5,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.51.1"
+  "version": "2.0.0"
```
Comment (fieldju) [inline lerna.json:8]: Let's revert this line.

The CI system sets this based off the tag that is created through the GH Releases page.

**Review by fieldju — CHANGES_REQUESTED** [review]
Everything LGTM except the manual setting of the Lerna version.
I assume the major version bump is from dropping node 6 from the Travis node list?

File: lerna.json (hunk 1 above)
Comment (fieldju) [reply lerna.json:8]: https://github.com/yaorg/node-measured/blob/master/scripts/publish.sh#L94

Comment (samarthmaniar) [reply lerna.json]: Thanks. Reverted the version back to 1.51.1

**Review by fieldju — APPROVED** [review]

**Pull request merged**